### PR TITLE
docs: promote cli command to install chrome stable

### PR DESCRIPTION
**Pull request by yury-s**
https://github.com/microsoft/playwright-java/issues/507

### CODE REVIEW

File: docs/src/browsers.md [hunk 1]
```diff
@@ -82,9 +82,27 @@ class Program
 }
 ```
 
-:::note
-Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use.
-:::
+Note that Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use. You can use Playwright command line interface to install the browsers:
```
Comment (mxschmitt) [inline docs/src/browsers.md:85]: Would be good to reference there to the dedicated CLI page like we do in the other places. (Search for command line)

File: docs/src/browsers.md [hunk 2]
```diff
@@ -82,9 +82,27 @@ class Program
...
+
+```bash js
+# Install Chrome stable
```
Comment (mxschmitt) [inline docs/src/browsers.md:88]: Looks similar to https://github.com/microsoft/playwright/blob/master/docs/src/cli.md#install-browsers which got recently added. Might make sense to dedup/reference?

Comment (yury-s) [reply docs/src/browsers.md:88]: Dunno, having it inline is convenient but let's keep a link for now.

File: docs/src/browsers.md (hunk 1 above)
Comment (yury-s) [reply docs/src/browsers.md:85]: done

**Review by mxschmitt — APPROVED** [review]

**Review by dgozman — APPROVED** [review]

**Pull request merged**